contrib/google.golang.org/grpc/grpc.go

```diff
@@ -31,6 +49,7 @@ func startSpanFromContext(
 		tracer.Tag(tagMethodName, method),
 		tracer.SpanType(ext.AppTypeRPC),
 	)
+
```

Comment:
Is this really needed?

Reply:
I didn't check the style guidance prior to submitting the PR. 
it's my personal preference for better legibility. I will remove the empty lines